modules/option-group/main.tf

```diff
@@ -0,0 +1,50 @@
+locals {
+  og_name = format("%s-%s-%s", var.name_prefix, var.engine, replace(var.engine_version, ".", "-"))
```

Comment:
I see a typo: `og_name` rather than `log_name`.

Reply:
No, it's supposed to be og_name. og meaning option group. No idea what a log_name would be in this context. I based that on pg_name from the parameter group module.

Reply:
That makes sense now that you explain it. I was thinking that you were trying to compete with me for being error prone today. :-D